composer.json

```diff
@@ -33,13 +32,11 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "MongoDB\\Tests\\": "tests/",
-            "MongoDB\\Benchmark\\": "benchmark/"
+            "MongoDB\\Tests\\": "tests/"
         },
         "files": [ "tests/PHPUnit/Functions.php" ]
     },
     "scripts": {
-        "benchmark": "phpbench run --report=aggregate",
```

Comment:
We could make this run a `composer update` in the benchmark directory and then invoke the benchmarking script from there. Just an idea to make this a little more accessible.

Reply:
Added